gammapy/modeling/tests/test_parameter.py

```diff
@@ -165,6 +172,49 @@ def test_parameters_to_table(pars):
     assert table["link"][0] == "test"
     assert table["link"][1] == ""
 
+    assert table["prior"][0] == ""
+    assert table["prior"][1] == "GaussianPrior"
+    assert table["type"][1] == ""
+
+    table.write(tmp_path / "test_parameters.fits")
+    Table.read(tmp_path / "test_parameters.fits")
```

Comment:
Should we add a least one test on the read table?

Reply:
Well there is no `from_table` method on `Parameters`. There is one on `DatasetModels` -> `update_parameter_from_table` but them it is implicitly tested in `test_set_parameters_from_table()` of `gammapy/modelling/models/tests/test_core.py`